src/quicer_nif.erl

```diff
@@ -156,8 +156,15 @@ open_lib() ->
             {error, _} ->
                 priv_dir()
         end,
+    LBMode =
+        case application:get_env(quicer, lb_mode, 0) of
+            ?QUICER_LOAD_BALANCING_IFIP_AS_SERVER_ID ->
+                lb_server_id(ipv4, application:get_env(quicer, lb_dev, ?DEFAULT_LB_DEV));
```

Comment:
Nit. It feels pretty bold to hardcode "default" interface, since it is largely unpredictable how it looks on different systems. Since it is an application-level env, might just make it explicit by making `lb_mode` follow the typespec `0..3 | {if_ip, IfName}` or something like that. What do you think?

Reply:
good point. To make it simpler I think I just use the `IfName::list()` instead of `{if_ip, IfName}`.